### PR TITLE
Added cat

### PR DESCRIPTION
**Pull request by RichardLitt**
Had some issues with attributes and body, but I think the use of headers clears it up

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -99,9 +99,17 @@ TODO
 # Group cat
 
 ## cat [GET /cat]
+Retrieves the object named by <ipfs-or-ipns-path> and outputs the data it contains.
+
++ Parameters
+    + arg (string, required) - The path to the IPFS object(s) to be outputted
 
 + Response 200 (text/plain)
```
Comment (dignifiedquire) [inline apiary.apib]: Not sure about the response time, as it depends on the file.

**Comment by dignifiedquire** [comment]
Would be nice to have an example response here for e.g. a textfile

File: apiary.apib (hunk 1 above)
Comment (RichardLitt) [inline apiary.apib]: Did I specify the response time?

Comment (dignifiedquire) [inline apiary.apib]: ..... response mime type...........

**Comment by dignifiedquire** [comment]
Merge conflicts

File: apiary.apib [hunk 2]
```diff
@@ -99,8 +99,29 @@ TODO
...
++ Response 200
+
+    + Headers
+        - Transfer-Encoding: chunked
+        - X-Stream-Output: 1
+
+    + Body
+
+    ```
+    $ curl -i http://localhost:5001/api/v0/cat?arg=QmeomffUNfmQy76CQGy9NdmqEnnHU9soCexBnGU3ezPHVH
```
Comment (dignifiedquire) [inline apiary.apib]: curl in the body?

Comment (RichardLitt) [inline apiary.apib]: Yeah. Not sure how to show the file we're using as the test? This is wrong; looking for a solution now.

**Comment by RichardLitt** [comment]
Removed example. Ready to merge. Opening issue to talk about it.

File: apiary.apib [hunk 3]
```diff
@@ -689,10 +689,17 @@ Import a tar file into ipfs
 
 + Response 200 (text/plain)
 
-## cat [GET /tar/cat]
```
Comment (dignifiedquire) [inline apiary.apib:692]: Why did you remove `/tar/cat`?

Comment (RichardLitt) [inline apiary.apib:692]: Good question. Rebase mess up. Removed cat in #1 by accident; added it back in properly now. Thanks for the catch. Clearly my bad.
